modules/cpms-changing-openstack-flavor-type.adoc

```diff
@@ -22,8 +22,10 @@ In {rh-openstack}, flavors define the compute, memory, and storage capacity of c
 providerSpec:
   value:
     ...
-    flavor: <1>
+    flavor: m1.xlarge <1>
 ----
 <1> Specify a larger {rh-openstack}  flavor type with the same base as the previous selection. For example, you can change `m6i.xlarge` to `m6i.2xlarge` or `m6i.4xlarge`.
```

Comment:
@EmilienM Must it be a larger one ?

Reply:
anything you want. The use case we should highlight is vertical scale. Where you use a bigger flavor or a smaller one is up to your needs.

Reply:
So maybe it's good to state that because from what I understood from the text it should be larger.

Reply:
https://github.com/openshift/openshift-docs/pull/66212/commits/67aff27738a7b6429f828d88be92f452c9573d99

Reply:
```suggestion
<1> Specify a {rh-openstack}  flavor type that has the same base as the existing selection. For example, you can change `m6i.xlarge` to `m6i.2xlarge` or `m6i.4xlarge`. You can choose larger or smaller flavors depending on your vertical scaling needs.
```

Reply:
https://github.com/openshift/openshift-docs/pull/66212/commits/5126856bb971640d64e8f432ad85349188fa81f8